src/lazyload-image.ts

```diff
@@ -5,20 +5,19 @@ import 'rxjs/add/operator/mergeMap';
 import 'rxjs/add/operator/catch';
 import 'rxjs/add/observable/of';
 import { Observable } from 'rxjs/Observable';
-import { getScrollListener } from './scroll-listener';
 
-function isVisible(element: HTMLElement, threshold = 0, _window = window) {
+function isVisible(element: HTMLElement, threshold = 0, window) {
```

Comment:
Yeah, sure. `_window` may be a bad name. But I think `window` is even worse (no offence) since it shadow a global variable. Which is confusing, especially for `window`: http://stackoverflow.com/a/11901566/4726243

Maybe: `windowRef` is a better name?